### PR TITLE
Mainnet Allocations

### PR DESCRIPTION
**Pull request by emilypi**

### CODE REVIEW

File: pact/genesis/mainnet/mainnet_coinbases.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+code: |-
+  (coin.coinbase "Emily" (keyset-ref-guard "Emily") 1000000.0)
```
Comment (fosskers) [inline pact/genesis/mainnet/mainnet_coinbases.yaml]: Was this just for debugging?

File: pact/genesis/mainnet/mainnet_keysets.yaml [hunk 2]
```diff
@@ -0,0 +1,171 @@
+code: |-
+  (define-keyset "Emily" (read-keyset "Emily"))
```
Comment (fosskers) [inline pact/genesis/mainnet/mainnet_keysets.yaml]: Here too.

File: pact/genesis/mainnet/mainnet_keysets.yaml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  (define-keyset "PS_C9" (read-keyset "PS_C9"))
+  (define-keyset "Coinlist Non-US Retail" (read-keyset "Coinlist Non-US Retail"))
+  (define-keyset "Coinlist Global Accredited" (read-keyset "Coinlist Global Accredited"))
+  (define-keyset "FTS_C1" (read-keyset "FTS_C1"))
+  (define-keyset "CS_C2" (read-keyset "CS_C2"))
+  (define-keyset "MM_C0" (read-keyset "MM_C0"))
+  (define-keyset "ST_C1" (read-keyset "ST_C1"))
+
+data:
+  "Emily": { keys: ["9a45ebe41b1ee9dc646f66f188582cf199da8e5f046b2b610b4884ec4f3e14d6"], pred: "keys-all" }
```
Comment (fosskers) [inline pact/genesis/mainnet/mainnet_keysets.yaml]: Here too.

File: rewards/mainnet_allocations.csv [hunk 4]
```diff
@@ -0,0 +1,1420 @@
+﻿Account Name,Time,Keyset Name,Quantity,Chain ID
```
Comment (fosskers) [inline rewards/mainnet_allocations.csv]: What's the difference between this file and the allocations YAML files?

File: rewards/mainnet_coinbase.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Coin Account,Keyset Ref Name,Quantity
+Emily,Emily,1000000
```
Comment (fosskers) [inline rewards/mainnet_coinbase.csv]: Debugging?

File: rewards/mainnet_keysets.csv [hunk 6]
```diff
@@ -0,0 +1,84 @@
+Keyset Name,"Predicate Function (keys-1, keys-2, keys-all)",Public Keys (as a Haskell List)
+Emily,keys-all,[9a45ebe41b1ee9dc646f66f188582cf199da8e5f046b2b610b4884ec4f3e14d6]
```
Comment (fosskers) [inline rewards/mainnet_keysets.csv]: Debugging?

File: rewards/mainnet_allocations.csv (hunk 4 above)
Comment (emilypi) [reply rewards/mainnet_allocations.csv]: this is the data used to generate the yamls

File: pact/genesis/mainnet/mainnet_keysets.yaml (hunk 3 above)
Comment (emilypi) [reply pact/genesis/mainnet/mainnet_keysets.yaml]: no we need this for tests

File: pact/genesis/mainnet/mainnet_keysets.yaml (hunk 2 above)
Comment (emilypi) [reply pact/genesis/mainnet/mainnet_keysets.yaml]: no we need this for tests

File: pact/genesis/mainnet/mainnet_coinbases.yaml (hunk 1 above)
Comment (emilypi) [reply pact/genesis/mainnet/mainnet_coinbases.yaml]: no we need this for tests

File: rewards/mainnet_coinbase.csv (hunk 5 above)
Comment (emilypi) [reply rewards/mainnet_coinbase.csv]: no we need this for tests

File: rewards/mainnet_keysets.csv (hunk 6 above)
Comment (emilypi) [reply rewards/mainnet_keysets.csv]: no we need this for tests

**Pull request merged**